src/components/fadedScrollView/fadedScrollView.api.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "FadedScrollView",
```

Comment:
FadedScrollView is an internal component. You can see that it's displayName = 'IGNORE'. Please remove api file for internal components